notebooks/adastra/data-owners/00-data-owners-create-domain.ipynb

```diff
@@ -8,21 +8,33 @@
     "# Create a Domain"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e8d79b0e",
+   "metadata": {},
+   "source": [
+    "**What is a Domain Node ?**<br>\n",
+    "A computer system (or collection of computer systems) which manages the remote study of a data owner's data by a data scientist. It is responsible for allowing the Data Owner to manage the data, as well as incoming requests from data scientists and for gatekeeping the data scientist's access to data, compute, and experimental results stored within the data owner's compute infrastructure.\n",
+    "\n",
+    "Let's start by creating our own Domain node. "
+   ]
+  },
   {
    "cell_type": "markdown",
    "id": "f4203778-938a-4af2-8aaa-ec126ef95cd1",
    "metadata": {},
    "source": [
     "## Option 1: Azure Marketplace\n",
-    "Create an [OpenMined PyGrid Domain](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/madhavajay1632269232059.openmined_mj_grid_domain_ubuntu_1?tab=Overview) node on the Azure Marketplace"
+    "\n",
+    "We can create an [OpenMined PyGrid Domain](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/madhavajay1632269232059.openmined_mj_grid_domain_ubuntu_1?tab=PlansAndPrice) node from the Azure Marketplace."
```

Comment:
MAYBE: When we click the link the in the azure market place, we could ensure the default version is 0.7.0 by itself, without users having to click and change.
Currently it shows 0.6.0 when we click the link

Reply:
Yep, I guess that's the plan when we launch a stable `0.7.0`, we will make it default.